scripts/docker_install.sh

```diff
@@ -42,6 +42,7 @@ apt-get install -y --no-install-recommends \
         libxkbcommon-x11-dev \
         libxrandr-dev
 
+npm install -g npm@8.3.0
```

Comment:
WDYT about adding this also to the macOS travis-ci.sh after `nvm install 14.15.4 # install this node version` so we have the same npm version?